content/en/templates/taxonomy-templates.md

```diff
@@ -137,9 +141,13 @@ Taxonomies can be ordered by either alphabetical key or by the number of content
 
 ```
 <ul>
-  {{ $data := .Data }}
+  {{ $type := .Type }}
   {{ range $key, $value := .Data.Terms.ByCount }}
-  <li><a href="{{ $.Site.LanguagePrefix }}/{{ $data.Plural }}/{{ $value.Name | urlize }}"> {{ $value.Name }} </a> {{ $value.Count }} </li>
+  {{ $name := .Name }}
+  {{ $count := .Count }}
+  {{ with $.Site.GetPage "taxonomy" (printf "%s/%s" $type $name) }}
```

Comment:
Same as said earlier, make it just one argument and use printf to create the argument with leading `/`.